### PR TITLE
Dashboard fix for md resolution, transactions load fix, block info row fix, address page issues, check mark issue

### PR DESCRIPTION
**Pull request by maxgrapps**
1) Fixed stats width for md resolution
Before:
Screen Shot 2019-07-03 at 18 22 33
After:
Screen Shot 2019-07-03 at 18 27 23

2) Transaction load animation fix on md resolution
Before:
Screen Shot 2019-07-03 at 18 31 54
After:
Screen Shot 2019-07-03 at 18 31 41

3) Block details row fix for sm resolutions
Before:
Screen Shot 2019-07-04 at 11 47 48
After:
Screen Shot 2019-07-04 at 11 57 19

4) Addresses tiles changed to table
Before:
Screen Shot 2019-07-04 at 12 18 23
After:
Screen Shot 2019-07-04 at 13 33 35

5) Address page card title issue on sm resolution
Before:
Screen Shot 2019-07-04 at 13 43 40
After:
Screen Shot 2019-07-04 at 13 49 13

6) Changed check mark color to white on active tab, Issue - https://github.com/poanetwork/blockscout/issues/1993
Before:
Screen Shot 2019-07-04 at 14 03 14
After:
Screen Shot 2019-07-04 at 14 04 50

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,7 @@
 - [#2266](https://github.com/poanetwork/blockscout/pull/2266) - allow excluding uncles from average block time calculation
 
 ### Fixes
+- [#2291](https://github.com/poanetwork/blockscout/pull/2291) - dashboard fix for md resolution, transactions load fix, block info row fix, addresses page issue
```
Comment (vbaranov) [inline CHANGELOG.md]: @maxgrapps incorrect place for CHANGELOG entry

Comment (maxgrapps) [reply CHANGELOG.md]: @vbaranov what is incorrect? Should I place something to features?

Comment (vbaranov) [reply CHANGELOG.md]: No, you are trying to change log of the published release

Comment (maxgrapps) [reply CHANGELOG.md]: @vbaranov got it, replaced to current version fixes, thanks

**Pull request merged**